tests/baselines/reference/api/typescript.d.ts

```diff
@@ -3450,7 +3450,6 @@ declare namespace ts {
             getLanguageService(): never;
             getHostForAutoImportProvider(): never;
             getProjectReferences(): readonly ts.ProjectReference[] | undefined;
-            getTypeAcquisition(): TypeAcquisition;
```

Comment:
Is this an intentional change?

Reply:
It isnt test only change but i had done this as part of https://github.com/microsoft/TypeScript/pull/56074 first and since it has no impact, left it there